pages/docs/output-data/rao-logs/example.md

```diff
@@ -0,0 +1,466 @@
+DEBUG c.f.farao.commons.logs.TechnicalLogs - Systematic sensitivity analysis [end]
+DEBUG c.f.farao.commons.logs.TechnicalLogs - Evaluated network action(s): Open line FR1-FR2, range actions have not been optimized, cost: -7.61 (functional: -7.61, virtual: 0.00)
+DEBUG c.f.farao.commons.logs.TechnicalLogs - Optimizing leaf...
+DEBUG c.f.farao.commons.logs.TechnicalLogs - Iteration 1: linear optimization [start]
+DEBUG c.f.farao.commons.logs.TechnicalLogs - Iteration 1: linear optimization [end]
+DEBUG c.f.farao.commons.logs.TechnicalLogs - Systematic sensitivity analysis [start]
+DEBUG c.f.farao.commons.logs.TechnicalLogs - Systematic sensitivity analysis [end]
+INFO  c.f.farao.commons.logs.TechnicalLogs - Iteration 1: better solution found with a cost of -224.34 (functional: -224.34)
+DEBUG c.f.farao.commons.logs.TechnicalLogs - Iteration 2: linear optimization [start]
+DEBUG c.f.farao.commons.logs.TechnicalLogs - Iteration 2: linear optimization [end]
+INFO  c.f.farao.commons.logs.TechnicalLogs - Iteration 2: same results as previous iterations, optimal solution found
+INFO  c.f.farao.commons.logs.TechnicalLogs - Optimized network action(s): Open line FR1-FR2, 1 range action(s) activated, cost: -224.34 (functional: -224.34, virtual: 0.00)
+INFO  c.f.farao.commons.logs.TechnicalLogs - Remaining leaves to evaluate: 0
+INFO  c.f.farao.commons.logs.TechnicalLogs - No better result found in search depth 1, exiting search tree
+INFO  c.f.farao.commons.logs.TechnicalLogs - Search-tree RAO completed with status DEFAULT
+INFO  c.f.farao.commons.logs.TechnicalLogs - Best leaf: Root leaf, 1 range action(s) activated, cost: -223.77 (functional: -223.77, virtual: 0.00)
+INFO  c.f.farao.commons.logs.TechnicalLogs - Best leaf: range action(s): PRA_CRA_PST_BE: 16
+INFO  c.f.farao.commons.logs.TechnicalLogs - Limiting element #01: margin = 223.77 MW, element NNL2AA1  BBE3AA1  1 at state Contingency_FR1_FR3 - curative, CNEC ID = "NL2-BE3-O - curative"
+INFO  c.f.f.commons.logs.RaoBusinessLogs - Scenario "Contingency_FR1_FR3": initial cost = -6.96 (functional: -6.96, virtual: 0.00), 1 range action(s) activated, cost after CRA = -223.77 (functional: -223.77, virtual: 0.00)
+INFO  c.f.farao.commons.logs.TechnicalLogs - Curative state Contingency_FR1_FR3 - curative has been optimized.
+INFO  c.f.farao.commons.logs.TechnicalLogs - Remaining post-contingency scenarios to optimize: 0
+INFO  c.f.f.commons.logs.RaoBusinessLogs - ----- Post-contingency perimeters optimization [end]
+~~~
+In this case, setting "PRA_CRA_PST_BE" to tap +16 produces the best result with a minimum margin, on this curative 
+perimeter, increased from +7 to +224 MW.
+
+## Second preventive RAO
+After optimising preventive remedial actions, simulating automatons, and optimising curative remedial actions, the 
+network situation is re-assessed after applying these actions on it.  
+Because the preventive remedial actions created a big virtual cost, a second preventive RAO is executed.  
```

Comment:
The second preventive is actually run because the 2P condition is "POSSIBLE_CURATIVE_IMPROVEMENT". FARAO doesn't see the virtual cost until the first loadflow pre 2P, which is computed after FARAO decides to run a 2P.